src/Hazelcast.Net/Clustering/HeartbeatOptions.cs

```diff
@@ -42,7 +41,7 @@ private HeartbeatOptions(HeartbeatOptions other)
         /// <para>Heartbeat will run periodically, and send a ping request to connections
         /// that have not been written to over the previous period.</para>
         /// </remarks>
-        public int PeriodMilliseconds { get; set; } = 5_000;
+        public int PeriodMilliseconds { get; set; } = 10_000;
```

Comment:
java client default is 5000 ms, can we use the same default value here?

Reply:
ok